common/gpu/intel/default.nix

```diff
@@ -21,6 +21,12 @@
         libvdpau-va-gl
         intel-media-driver
       ];
+
+      hardware.opengl.extraPackages32 = with pkgs.driversi686Linux; [
+        (if (lib.versionOlder (lib.versions.majorMinor lib.version) "23.11") then vaapiIntel else intel-vaapi-driver)
+        libvdpau-va-gl
```

Comment:
Do you have an idea in which scenarios libvdpau-va-gl is useful?

Reply:
Not sure.  I just copied the list of packages already being added to `.extraPackages`.

Reply:
I guess it's fine because it's only used if you have `opengl.driSupport32Bit` enabled?

Reply:
We can still remove it, if it causes problems down the line.